charts/wildfly/README.md

```diff
@@ -175,6 +175,8 @@ If the Helm chart is only used to build the application image, you can skip the
 | ----- | ----------- | ------- | ---------------------- |
 | `deploy.annotations` | Map of `string` annotations that are applied to the deployment and its pod's `template` | - | [Kubernetes documentation](https://kubernetes.io/docs/concepts/overview/working-with-objects/annotations/) |
 | `deploy.enabled` | Determines if deployment-related resources should be created. | `true` | Set this to `false` if you do not want to deploy an application image built by this chart. |
+| `deploy.argsOverride` | An `args` array to override the default arguments of the container's command. | - | [Kubernetes documentation](https://kubernetes.io/docs/tasks/inject-data-application/define-command-argument-container/#changing-commands-and-arguments-that-are-run-in-a-container). |
+| `deploy.commandOverride` | A `command` array to override the default command of the container.  | -  | [Kubernetes documentation](https://kubernetes.io/docs/tasks/inject-data-application/define-command-argument-container/#changing-commands-and-arguments-that-are-run-in-a-container). |
```

Comment:
doc is missing for `deploy.securityContext` with a link to https://kubernetes.io/docs/tasks/configure-pod-container/security-context/